resources/welcome/index.html

```diff
@@ -169,61 +162,409 @@ <h3 id="generate-app">Generate Azure IoT application</h3>
+                </div>
+                <div>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-tools/issues">Issues</a>
                 </div>
                 <div>
-                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit">Repository</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-tools/blob/master/CHANGELOG.md">Changelog</a>
                 </div>
+            </div>
+            <div>
+                <h3>Repositories</h3>
                 <div>
-                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit/issues">Issues</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-tools">Azure IoT Tools</a>
                 </div>
                 <div>
-                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit/wiki">Wiki</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit">Azure IoT Hub Toolkit</a>
                 </div>
                 <div>
-                    <a href="https://gitter.im/Microsoft/azure-iot-toolkit">Gitter</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-edge">Azure IoT Edge</a>
                 </div>
                 <div>
-                    <a href="https://marketplace.visualstudio.com/items/vsciot-vscode.azure-iot-toolkit/changelog">Changelog</a>
+                    <a href="https://github.com/Microsoft/vscode-iot-workbench">Azure IoT Device Workbench</a>
                 </div>
             </div>
             <div id="resources">
                 <h3>Resources</h3>
                 <div>
-                    <a href="https://channel9.msdn.com/Shows/Internet-of-Things-Show/Azure-IoT-Toolkit-extension-for-Visual-Studio-Code">Channel 9 video: Walkthrough of Azure IoT Hub Toolkit extension</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit/wiki">Azure IoT Hub Toolkit Documentation</a>
                 </div>
                 <div>
-                    <a href="https://blogs.msdn.microsoft.com/iotdev/2018/08/08/quickly-build-your-azure-iot-application-in-vs-code-with-node-js-python-or-rest-api/">Quickly build your Azure IoT application with Node.js, Python or REST API</a>
+                    <a href="https://github.com/Microsoft/vscode-azure-iot-edge/blob/master/README.md">Azure IoT Edge Documentation</a>
                 </div>
                 <div>
-                    <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit/wiki/Quickstart-Node.js">Create and control an IoT device connected to an IoT hub (Node.js)</a>
+                    <a href="https://github.com/Microsoft/vscode-iot-workbench/blob/master/README.md">Azure IoT Device Workbench Documentation</a>
+                </div>    
+                <div>
+                    <a href="https://aka.ms/iot-devkit">IoT Devkit Home Page</a>
```

Comment:
It is for vs code extension, do we want to include device?

Reply:
Liya recommended we put this links in our extension